### PR TITLE
Make attachments only appear if supported

### PR DESCRIPTION
**Pull request by rgoessl**
Don't want attachments in right panel for COA eval.
Goes along with this API change: https://github.com/Brightspace/lms/pull/2746

### CODE REVIEW

File: components/controllers/ConsistentEvaluationHrefController.js [hunk 1]
```diff
@@ -106,10 +106,10 @@ export class ConsistentEvaluationHrefController {
 					}
 				}
 			}
-		}
 
-		if (root.hasSubEntityByRel(editSpecialAccessApplicationRel)) {
-			specialAccessHref = root.getSubEntityByRel(editSpecialAccessApplicationRel).properties.path;
+			if (root.hasSubEntityByRel(editSpecialAccessApplicationRel)) {
```
Comment (rgoessl) [inline components/controllers/ConsistentEvaluationHrefController.js:110]: Unrelated fix. Ensure root is not null

**Review by Callum-Mitchell — APPROVED** [review]

**Review by jeroach — APPROVED** [review]
